src/Organic/Organic.php

```diff
@@ -447,11 +474,11 @@ public function useAffiliate() : bool {
      * @return bool
      */
     public function useAmp() : bool {
-        return $this->isEnabledAndConfigured() && $this->ampEnabled;
+        return $this->useAds() && $this->ampEnabled;
```

Comment:
Affiliate has AMP integration too
Overall I considered AMP and Prefill an plugin-level features, not something related to Ads or Affiliate

Reply:
When is prefill used outside of Ads?